src/test/java/seedu/duke/command/TagCommandTest.java

```diff
@@ -7,8 +7,4 @@
     @Test
     void execute() {
```

Comment:
If it is a test method should we not follow the following naming conventions?

`whatIsBeingTested_descriptionOfTestInputs_expectedOutcome` 
OR
`testMethodName`

Reply:
yup it should! I just generate a template test class therefore it consists of the execute function inside. All the test functions should be renamed accordingly.

Reply:
He haven't start on it. He created the classes as a template first I think.